partials/method.handlebars

```diff
@@ -90,20 +90,18 @@
       </div>
     {{/if}}
 
-    {{#if throws}}
-      <div class="return">
+    {{#throws}}
```

Comment:
It seems it's not. If I try to declare multiple @throws, I only get the last one and, even iterating `throws` object with an `#each` gives me an unique 0 index.